src/ActiveLogin.Authentication.BankId.AspNetCore.Azure/KeyVault/AzureKeyVaultCertificateClient.cs

```diff
@@ -65,6 +66,21 @@ private AzureKeyVaultCertificateClient(KeyVaultClient client)
         }
 
         public async Task<X509Certificate2> GetX509Certificate2Async(string keyVaultSecretIdentifier)
```

Comment:
Would be nice to rename this to: `keyVaultIdentifier`, cause we support both secret and cert.
Also, we should then rename the other properties, for example in: `ClientCertificateFromAzureKeyVaultOptions`

This will cause a breaking change, but we can make next release a new major version. 
For `ClientCertificateFromAzureKeyVaultOptions.AzureKeyVaultSecretIdentifier` we would rename that, but create a (deprecated) set only prop with the old name, bot to break to many applications...